x-pack/metricbeat/module/googlecloud/billing/_meta/data.json

```diff
@@ -0,0 +1,30 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "cloud": {
+        "account": {
+            "id": "elastic-bi",
+            "name": "elastic-bi"
+        },
+        "provider": "googlecloud"
+    },
+    "event": {
+        "dataset": "googlecloud.billing",
+        "duration": 115000,
+        "module": "googlecloud"
+    },
+    "googlecloud": {
+        "billing": {
+            "cost_type": "regular",
+            "invoice_month": "202008",
```

Comment:
Would it make sense to convert this to a timestamp?

Reply:
Hmm I think we should keep it as a string to match `invoice.month` in billing table schema from BigQuery: https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#data-schema